content/rc/subscriptions/connectivity/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: Networking and Connectivity
+linkTitle: Connectivity
+description: 
+weight: $weight
+alwaysopen: false
+toc: "true"
+categories: ["RC"]
+aliases: 
+---
+
+Redis offers a few ways you can securely connect your application and your database without exposing your application's [virtual private cloud](https://en.wikipedia.org/wiki/Virtual_private_cloud).
+
+## [Private Service Connect]({{<relref "/rc/subscriptions/connectivity/private-service-connect">}})
+
+[Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) (PSC) creates a private endpoint that allows secure connections to Redis Cloud databases without exposing your application's VPC.
+
+Private Service Connect is only available for Flexible subscriptions hosted on Google Cloud.
```

Comment:
```suggestion
Private Service Connect is only available for Flexible subscriptions hosted on Google Cloud.
```

Should we make "Flexible subscriptions" a link to the subscription description page?